### ISSUE TITLE
[Bug] Master branch types break example

### ISSUE THREAD

**Issue by PupoSDC** [Camberi/firecms#82]
I think this issue is only available on master and its potentially not release as of now, but when I install and link the example project to master I get these warnings:

```ts
        categories: {
            title: "Categories",
            validation: { required: true },
           // Type '"array"' is not assignable to type '"timestamp"'.
            dataType: "array",
            of: {
               /** */
            }
        },
        publisher: {
            title: "Publisher",
            description: "This is an example of a map property",
            // Type '"map"' is not assignable to type '"timestamp"'.
            dataType: "map",
            properties: {
                  /** */
            }
        },
```

**Comment by fgatti675** [comment]
Hi @PupoSDC 
I have done some dependencies update and revamp in the master branch 
Related to what we were discussing in https://github.com/Camberi/firecms/issues/80 I have moved all main peer dependencies in the example to use the `..link` system instead of needing to link them manually, since it feels more reliable:
https://github.com/Camberi/firecms/blob/master/example/package.json
Also MUI dependencies have been declared peer dependencies (as they were meant to be, but weren't incorrectly)
I am not getting any type errors locally. 
Could you get the latest `master` version, do a full clean of `node_modules` and possible `yarn.lock` in your project and see if this issues get resolved?

**Comment by PupoSDC** [comment]
will give it a try ASAP!

**Comment by fgatti675** [comment]
This changes have all been merged and released in version 0.39.0 :)
